docker/build_image.sh

```diff
@@ -11,6 +11,7 @@ CUDA_VERSION=""
 UBUNTU_VERSION="ubuntu:20.04"
 USE_LOCAL_SERVE_FOLDER=false
 BUILD_WITH_IPEX=false
+PYTHON_VERSION=3.8
```

Comment:
Seems like default is 3.8 and not 3.9 like docs indicate? Don't mind either way let's just stay consistent

Reply:
Resolved. Thanks. I had not push the latest update